deps/wazuh_testing/wazuh_testing/modules/fim/event_monitor.py

```diff
@@ -58,3 +66,72 @@ def callback_detect_event(line):
             return json_event
     except (json.JSONDecodeError, AttributeError, KeyError) as e:
         logger.warning(f"Couldn't load a log line into json object. Reason {e}")
+
+
+def callback_detect_end_scan(line):
+    """ Callback that detects if a line in a log is a scan_end event
+    Args:
+        line (String): string line to be checked by callback. Passed in by File_Monitor.
+    """
+    msg = CB_FIM_EVENT
+    match = re.match(msg, line)
+    if not match:
+        return None
+
+    try:
+        if json.loads(match.group(1))['type'] == 'scan_end':
+            return True
+    except (JSONDecodeError, AttributeError, KeyError) as e:
+        logger.warning(f"Couldn't load a log line into json object. Reason {e}")
+
+
+def callback_num_inotify_watches(line):
+    """ Callback that detects if a line contains the folders monitored in realtime event
+    
+    Args:
+        line (String): string line to be checked by callback. Passed in by File_Monitor.
+    """
+    match = re.match(CB_REALTIME_MONITORED_FOLDERS, line)
+
+    if match:
+        return match.group(1)
+
+
+def callback_real_time_whodata_started(line):
+    """ Callback that detects if a line contains "Whodata engine started" event
+    Args:
+        line (String): string line to be checked by callback. Passed in by File_Monitor.

```

Comment:
I don't see that adding ` Passed in by File_Monitor` clarifies something.

Reply:
same as above